js/src/3rdparty/sms-verification/index.js

```diff
@@ -15,17 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import { stringify } from 'querystring';
-import React from 'react';
-
-export const termsOfService = (
```

Comment:
Very weird indeed, since the 3rdparty libraries are also stuff that we would like to start publishing under the `@parity/` namespace. Unless we make an exception for the specific one and don't publish it at all.